resultstore-search-client/src/utils/index.ts

```diff
@@ -0,0 +1,18 @@
+/*
```

Comment:
Just wondering about the naming: why is this called `index.ts` instead of `utils.ts`?